lib/cli.js

```diff
@@ -137,6 +137,18 @@ const cli = {
 
             log.info("v" + require("../package.json").version);
 
+        } else if (currentOptions.printConfig) {
```

Comment:
Yeah, I thought about that. You are right, the behavior has changed. So I can add that check back in. (I removed a unit test for that specific behavior. I guess I hadn't considered that use case important, oops.)

I'll add a check and I'll get that test back into tests/lib/cli.js. Sound good?